src/Components/OptionCardGroup/OptionCardGroup.js

```diff
@@ -48,23 +41,43 @@ const OptionCardGroup = ({ stateVariable, options, errorController }) => {
           }}
         >
           <Card
-            className={state[stateVariable][optionKey] ? 'selected-option-card' : 'unselected-option-card'}
-            sx={{ width: '11.25rem', height: '11.25rem', display: 'grid', placeItems: 'center' }}
+            className={memberData[stateVariable][optionKey] ? 'selected-option-card' : 'unselected-option-card'}
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              border: '.05px solid #6D6E71',
+              boxShadow: '0px 4px 4px 0px rgba(0,0,0,0.25)',
+              height: '15rem',
+              ':hover': {
+                backgroundColor: '#ECDEED',
+                color: '#1d1c1e',
+              },
+            }}
```

Comment:
I feel like this is a lot of CSS to inline. What do you think about adding a class to encapsulate all of this CSS, seeing that we already are using classes to mark whether or not it is selected?